authorization_services/topics/hello-world-create-realm.adoc

```diff
@@ -1,21 +1,26 @@
 [[_getting_started_hello_world_create_realm]]
-= Creating a Realm and a User
+= Creating a realm and a user
 
 The first step in this tutorial is to create a realm and a user in that realm. Then, within the realm we will create a single client application, which then becomes a <<_overview_terminology, resource server>> for which you need to enable authorization services.
 
-To create a realm and a user complete the following steps:
+.Procedure
 
 . Create a realm with a name *hello-world-authz*. Once created, a page similar to the following is displayed:
 +
 .Realm hello-world-authz
 image:{project_images}/getting-started/hello-world/create-realm.png[alt="Realm hello-world-authz"]
 
-. Create a user for your newly created realm. Click *Users*. The user list page opens.
+. Click *Users*.
++
+The user list page displays where you can create a user.
+
+. Click *Add User*.
+
+. Complete the *Username*, *Email*, *First Name*, and *Last Name* fields.
 
-. On the right side of the empty user list, click *Add User*.
+. Toggle *User Enabled* to *ON*
 
-. To create a new user, complete the *Username*, *Email*, *First Name*, and *Last Name* fields.
-Click the *User Enabled* switch to *On*, and then click *Save*.
+. Click *Save*.
 +
 .Add User
 image:{project_images}/getting-started/hello-world/create-user.png[alt="Add User"]
```

Comment:
@andymunro : The casing of image heading is all caps throughout this guide. Does it needs to match the other headings? Please check and if required then it needs to be changed throughout.

Reply:
Thanks.  You are right, @nehachopra27.  I will change it to sentence case, meaning lower case except for the first word or a proper name.